VirtoCommerce.CatalogModule.Data/ExportImport/TabularProduct.cs

```diff
@@ -0,0 +1,63 @@
+using System;
```

Comment:
Please move it to Catalog.Core (as here https://github.com/VirtoCommerce/vc-platform-core/commit/0429ecf54bd651826061cff8cb0adcf613e6b38f)